test/claims-status/components/ClaimsStatusApp.unit.spec.jsx

```diff
@@ -20,10 +20,11 @@ describe('<ClaimsStatusApp>', () => {
     expect(tree.subTree('RequiredLoginView').props.authRequired).to.equal(3);
   });
   it('should render children', () => {
+    const props = { isDataAvailable: true };
     const tree = SkinDeep.shallowRender(
       <AppContent
         available
-        authorized>
+        authorized {...props}>
```

Comment:
Since `props` only has one property in it, could just have:

```
<AppContent available authorized isDataAvailable>
```

In which case, there is a case for renaming that prop name...